### PR TITLE
New package: blender

### PR DESCRIPTION
**Pull request by qwertos**
I'm not super happy with this yet, but I'm getting to the point where some feedback might be helpful.

### CODE REVIEW

File: var/spack/repos/builtin/packages/blender/package.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/blender/package.py:1]:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/blender/package.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    depends_on('python@3.7.7:', when="@2.92.0:")
+
+    depends_on('py-numpy', when="@2.80:")
+    depends_on('py-numpy@1.17.5:', when='@2.92.0:')
+    
+    depends_on('glew')
+    depends_on('glew@1.13.0:', when='@2.92.0:')
+
+    #depends_on('opengl')
+    # depends_on('openglu')
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/blender/package.py:47]: Is there a reason to retain commented-out dependencies here and below?

**Review by tldahlgren — CHANGES_REQUESTED** [review]

**Comment by tldahlgren** [comment]
Oops.  Should've waited until after you removed "Draft".

**Comment by dylex** [comment]
Thanks for putting this together -- it'd be great to see it get in.

I know you're still working on it, but in case it's helpful, this fails for me when using `intel-oneapi-tbb` (rather than `intel-tbb`) -- I think this might be a blender limitation that should just have a conflict or something.

It also fails with the (default) `~ffmpeg`, as it tries to build the ffmpeg plugin anyway.  I think it needs an explicit `WITH_CODEC_FFMPEG=OFF` for this.

Build 3 worked fine.

**Comment by alalazo** [comment]
@qwertos Is this still a work in progress?

**Comment by qwertos** [comment]
Not active. I'm gonna close and resubmit when I pick this up again.

**Pull request closed without merging**